.env

```diff
@@ -1 +1,7 @@
-DB_URI="postgresql+psycopg2://postgres:pw@localhost:5432/postgres"
+MPI_DB_TYPE=postgres
```

Comment:
@cbrinson-rise8 these look like the old settings, I think you may have had a merge resolution issue.  This file should just be 1 line.  https://github.com/CDCgov/RecordLinker/blob/main/.env

Reply:
good catch! Changed it back to be the same as the .env on main